src/main/java/eu/bitwalker/useragentutils/OperatingSystem.java

```diff
@@ -122,7 +122,15 @@
 		iOS6_IPAD(	Manufacturer.APPLE, OperatingSystem.MAC_OS_X_IPAD, 51, "iOS 6 (iPad)", new String[] { "OS 6" },  null, DeviceType.TABLET, null ), // before Mac OS X
 		MAC_OS_X_IPHONE(Manufacturer.APPLE, OperatingSystem.IOS, 40, "Mac OS X (iPhone)", new String[] { "iPhone" },  null, DeviceType.MOBILE, null ), // before Mac OS X
 		MAC_OS_X_IPOD(	Manufacturer.APPLE, OperatingSystem.IOS, 30, "Mac OS X (iPod)", new String[] { "iPod" },  null, DeviceType.MOBILE, null ), // before Mac OS X
-	
+	iOS8_4(	Manufacturer.APPLE, null, 101, "iOS 8.4", new String[] { "CFNetwork/711.4.6" },  null, DeviceType.MOBILE, null ), // use CFNetwork
```

Comment:
Actually, we saw lots of iOS apps have user agent string like "appName/version.number CFNetwork/version.number Drawin/version.number".  If the app accesses a web-service, it will use such user agent string. I can't tell it MOBILE or TABLET because of lack of enough information.

Also, we can't make them to be the children of iOS, because its user agent string doesn't contain "iphone os" or "like mac os x". So I have to indicate its parent to be null.